pkg/specgen/namespaces_test.go

```diff
@@ -161,7 +161,14 @@ func TestParseNetworkFlag(t *testing.T) {
 			name:   "bridge mode with invalid option",
 			args:   []string{"bridge:abc=123"},
 			nsmode: Namespace{NSMode: Bridge},
-			err:    "unknown bridge network option: abc",
+			networks: map[string]types.PerNetworkOptions{
```

Comment:
can you rename the test to not say invalid but rather unknown.

Also maybe add another case where add 2+ options to make sure it is working for multiple keys. I know it does as the code is simple enough but better to validate that